### ISSUE TITLE
Docs: GitHub tip

### ISSUE THREAD

**Issue by johnjacobkenny** [twindle-co/twindle#167]
Here is a small trick. If you use a supported keyword in the pull request's description like:

```
close, closes, closed, fix, fixes, fixed, resolve, resolves, resolved
```

**Example**: Closes #118

Then the linked issue will be auto closed when you merge the commit into the default branch.

For more info:

- [Linking a pull request to an issue](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [Automatically closing issue from pull request in GitHub](https://stackoverflow.com/questions/12235620/automatically-closing-issue-from-pull-request-in-github).

_Originally posted by @arthurbuhl in https://github.com/twindle-co/twindle/pull/124#issuecomment-709196486_

**Comment by arthurbuhl** [comment]
## Proof Of Concept

### PR

A new PR (#169) has been created where description is like: `close #168` 

### Linked Issue

The issue (#168) has now been linked to this PR and will be closed automatically once the PR is merged.

**Comment by piratelicorne** [comment]
This was referenced in the FAQ document and updated in the PR #201  ✅

**Comment by proful** [comment]
@johnjacobkenny documentation related thing can be moved to appropriate file under docs & this issue can be closed.

**Comment by johnjacobkenny** [comment]
Thanks @piratelicorne
